### ISSUE TITLE
Starry Bee broken

### ISSUE THREAD

**Issue by winggndm** [AllTheMods/ATM-6#2309]
The Apiary sizes have been changed and has broken the Starry Bee.  The bee no longer goes inside.  The size used to be 7x6x7 and now they have to be 7x5x7.

**Comment by ladylexxie** [comment]
The side of the apiary is exactly the same. The only thing that changed is that it doesn't detect the floor to be valid anymore.

**Comment by ClintRajaniemi** [comment]
I found this issue on ATM6 1.8.3. What I found was that placing the bee in the Apiary multiblock, it would find its flower the first time and then head toward the Apiary block. It might get in the first time, but after that, it would end up in the top corner of the Apiary multiblock and just stay there. It looked like it was trying to find a path to the Apiary block, but would just turn.

I wondered if it had to do with the hit box (or something along those lines) given that this bee had a larger size modifier. I changed it to 1 and it no longer had problems getting into the Apiary block.

File changed: config/resourcefulbees/bees/special/Starry.json
Value changed to: "sizeModifier": 1,

I don't know if it's an issue on later versions or if there are other issues this change might bring up, but I didn't have any problems with it.
